index.html

```diff
@@ -86,6 +86,18 @@ <h3 class="ba-skill__title">dreams.</h3>
         </div>
 
     </main>
+    <footer>
+        <div class="footer__wraper">
+            <form action="">
+                <input type="text" placeholder="Emaile">
```

Comment:
This is not important, but I think, there is a mistake in placeholder text. Rewrite, please.

Reply:
Done. Please check. Is everything all right now?